scripts/tmpl/get.kobman.io.tmpl

```diff
@@ -0,0 +1,262 @@
+
+#!/bin/bash
+
+#Install: stable
+
+# Global variables
+KOBMAN_VERSION="@KOB_VERSION@"
+KOB_ARCHIVE_DOWLOAD_REPO="@KOB_ARCHIVE_DOWLOAD_REPO@"
+KOBMAN_PLATFORM=$(uname)
+KOBMAN_SERVICE="https://raw.githubusercontent.com"
```

Comment:
Mention what service this is based on end point. This must point to the gh-pages and should have the namespace